client/src/utils/googleCalenderEvents.js

```diff
@@ -130,6 +131,7 @@ export const deleteEvent = (eventId) => {
         });
 };
 
-gapi.load('client:auth2', function () {
-    gapi.auth2.init({ client_id: GOOGLE_CLIENT_ID });
-});
+//This is where you load the modal with google auth, you don't need this, this is for getting someones credentials and use them to log into your app.

```

Comment:
Disabled popup. It's a bug, you don't need the user to log in with their accounts in order to schedule an event on the calendar.